blade/procData/models.py

```diff
@@ -11,4 +11,17 @@ class Meta:
 		ordering = ['-id']
 
 	def __str__(self):
-		return self.email
\ No newline at end of file
+		return self.email
+
+
+#class results
+
+#class query
+class dbQueriedUsers(models.Model):
+	user = models.CharField(max_length=100,unique=True)
```

Comment:
only username, no email?
could also add an `added` column to track when the record was created

Reply:
`added = models.DateTimeField(auto_now_add=True)` would auto-set the date when creating a new row, so then you don't have to specify it in the objects.create call